{{ cookiecutter.repo_name }}/.envrc

```diff
@@ -1,16 +1,8 @@
 #!/bin/bash
 [ -f .env ] && source .env
 
+# export S3_INPUT_PATH="s3://nesta-ds-projects/<MISSION OR PRACTICE>/<PROJECT NAME>"
 
-export BUCKET="{{ cookiecutter.s3_bucket }}"
```

Comment:
Maybe this speaks to me not using my project's S3 bucket as it is intended - but I have been storing both inputs and outputs on S3 (I have been using it to store all my models, output datasets..) and therefore use BUCKET quite a lot when I'm saving out things.

So either I need to stop saving outputs to S3 (although, I wonder where else and what are the reasons for this?), or does the `S3_INPUT_PATH` variable name need to change to something like `S3_PATH`, and `inputs-pull` needs to go back to `aws s3 sync ${S3_PATH}/inputs inputs`?

Reply:
There's nothing to stop you from having something like the following in your `.envrc` and using `S3_PROJECT_PATH` to save outputs to S3...

```bash
export S3_PROJECT_PATH="s3://nesta-ds-projects/dap/awesome-liz-stuff"
export S3_INPUT_PATH="${S3_PROJECT_PATH}/inputs"
```

My (personal) recommendation (not a requirement) would be to use Metaflow and the S3 datastore for outputs.


I've opted to not "pick a side" within the cookiecutter when it comes to outputs because I can't make a general recommendation that works for everyone.